bgcval2/custom_plots/ensemble_plots.py

```diff
@@ -106,7 +106,7 @@ def getAMOCdata(j, field='AMOC'):
 
     if not os.path.exists(fn):
         print('Does not exist:', fn)
-    shelve = shopen(fn)
+    shelve = shOpen(fn)
```

Comment:
here too

Reply:
Not worth the effort - don't use this code.